Dockerfile

```diff
@@ -1,26 +1,26 @@
-FROM alpine:3.7
+FROM python:3.8.2-slim
 LABEL maintainer="smartlab-dev@mpt.mp.br"
 
+ENV PYTHONPATH /app:/usr/lib/python3.8/site-packages
+
 COPY requirements.txt /app/requirements.txt
 COPY app/*.py /app/
 COPY uwsgi.ini /etc/uwsgi/conf.d/
 COPY start.sh /start.sh
 
-RUN apk --update --no-cache add build-base libffi-dev openssl-dev python3-dev libffi openssl ca-certificates python3 cyrus-sasl-dev libstdc++ uwsgi-python3 gfortran openblas-dev && \
-    pip3 install --upgrade pip setuptools && \
+WORKDIR /app
```

Comment:
I think you can move this near to the end, if you need to replace it, you won't need to rebuild all stages again.